header-footer-grid/assets/scss/frontend/components/_palette-switch.scss

```diff
@@ -0,0 +1,30 @@
+.toggle-palette {
```

Comment:
this needs to be added as inline style, minified. font size, line height, letter spacing should be based on the main menu used.

Reply:
@JohnPixle how about here, should we use the font settings of the label as the font settings of the menu?

Reply:
I've added the class that is being used for the primary navigation for the label so that it will use the same settings and removed the css font styles. Also added the whole style as inline minified, instead of having it compile like some of the components.